GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/Keywords.cshtml

```diff
@@ -1,11 +1,8 @@
 ﻿@model GenderPayGap.WebUI.Models.Search.SearchViewModel
 <div id="SearchByKeywordsGroup" class="form-group" style="position: relative;">
-    <h1 class="heading-large">
-        Search by <span class="visuallyhidden">either employer name or SIC code</span>
-    </h1>
     <fieldset class="inline">
-        <legend class="visually-hidden">
-            Select how you want to search: either by employer name or by Sector (SIC code)
+        <legend class="heading-large">
+            Search by <span class="visuallyhidden">either employer name or SIC code</span>

```

Comment:
- I wasn't sure whether their wording meant that they actually wanted an `h1` inside the `legend`, or to move the copy from one to the other? Happy to wrap in an `h1` if we think that's what they meant though - although for a screen-reader I imagine an h1 inside a legend could be misleading since it's not actually a full page header?
- I read that screenshot as displaying the values on the two radio buttons below this legend - I didn't think it meant to remove that visually hidden text. May be wrong though? Might leave this for now and get Sarah's take on it tomorrow :)

Reply:
It's really important to have an `<h1>` on each page (which tends to win on the "is X or Y more important for accessibility"!)
They actually use `<h1>` inside `<legend>` reasonably often (I hadn't seen it before, but apparently it's ok)
e.g. the first HTML code snippet on https://design-system.service.gov.uk/components/radios/

Reply:
I assumed we'd be removing `either employer name or SIC code` because of the difference between their two screenshots (but maybe worth checking with DAC)

Reply:
On a re-read I agree that this is what they're wanting here!